operator/pkg/util/util.go

```diff
@@ -23,6 +24,14 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const (
+	manifestDir                     = "manifest"
```

Comment:
I think this should be dynamic with a flag and documented so one can create a volume binding for this path (e.g. when you want a PVC for these manifests)

Reply:
Configuration makes total sense, but the issue roots deeper than only "manifest" in this case. 
I formulated this requirement in an [issue](https://github.com/kyma-project/module-manager/issues/142) which extends beyond the scope of this PR.